python/test/test_datasetslice.py

```diff
@@ -49,5 +53,16 @@ def test_slice_single_index(self):
                          self._d[Data.Value, "a"][Dim.X, 7].numpy)
 
 
+    def test_range_based_slice(self):
+        subset = slice(1,4,1)
+        # Create slice
+        ds_slice = self._d[Dim.X,subset]
+        # Test via variable_slice
+        self.assertEquals(len(ds_slice[Coord.X]), len(range(subset.start, subset.stop, subset.step)))
+
+    def test_repr(self):
```

Comment:
I would avoid putting such tests, since this is likely to change. The pretty formatting is not something that is promised to stay as it is.

Reply:
Agreed